tao/tao_bridge/ral/BUILD

```diff
@@ -0,0 +1,82 @@
+package(default_visibility = ["//visibility:public"])
+load("@com_google_protobuf//:protobuf.bzl", "cc_proto_library")
+load("@local_config_tf//:build_defs.bzl", "tf_copts")
+load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda", "cuda_library")
+
+cc_proto_library(
+    name = "compile_metadata",
+    srcs = [
+        "tensorflow/compiler/mlir/xla/compile_metadata.proto",
+    ],
+)
+
+filegroup(
+    name = "ral_headers",
+    srcs = glob(["tensorflow/**/*.h"]),
+)
+
+cc_library(
+    name = "bridge_ral",
+    srcs = [
+        "tensorflow/compiler/mlir/xla/compile_metadata.pb.cc",
+        "tensorflow/compiler/mlir/xla/ral/context/common_context_impl.cc",
+        "tensorflow/compiler/mlir/xla/ral/context/stream_executor_based_impl.cc",
+        "tensorflow/compiler/mlir/xla/ral/context/tensorflow/tf_context_impl.cc",
+        "tensorflow/compiler/mlir/xla/ral/context/tensorflow/tf_kernel_impl.cc",
+        "tensorflow/compiler/mlir/xla/ral/device/cpu/cpu_driver.cc",
+        "tensorflow/compiler/mlir/xla/ral/device/gpu/gpu_driver.cc",
+        "tensorflow/compiler/mlir/xla/ral/ral_api.cc",
+        "tensorflow/compiler/mlir/xla/ral/ral_context.cc",
+        "tensorflow/compiler/mlir/xla/ral/ral_helper.cc",
+        "tensorflow/compiler/mlir/xla/ral/ral_logging.cc",
+    ],
+    hdrs = [
+        ":ral_headers",
+    ],
+    copts = [
+        "-DDISC_BUILD_FROM_TF_BRIDGE",
+        "-Iexternal/org_tao/tao_bridge/ral",
```

Comment:
this include path looks wired...

Reply:
This is what I want to discuss. 
It works under current directory tree, however a more general solution is needed.
I see the bazel build dir, the build command is executed under org_tf_blade dir, the ral code is in the path of org_tf_blade/external/org_tao/tao_bridge/ral/... , thus the strange include path.

Reply:
include paths are expected to be carried via target dependency.